packages/beacon-node/src/api/impl/validator/index.ts

```diff
@@ -1128,15 +1132,21 @@ export function getValidatorApi({
             );
             await network.publishContributionAndProof(contributionAndProof);
           } catch (e) {
+            const logCtx = {
+              slot: contributionAndProof.message.contribution.slot,
+              subcommitteeIndex: contributionAndProof.message.contribution.subcommitteeIndex,
+            };
+
+            if (
+              e instanceof SyncCommitteeError &&
+              e.type.code === SyncCommitteeErrorCode.SYNC_COMMITTEE_AGGREGATOR_ALREADY_KNOWN
+            ) {
+              logger.debug("Ignoring known contributionAndProof", logCtx);
```

Comment:
This error was already ignored previously (added in https://github.com/ChainSafe/lodestar/pull/2598) but it got removed again by https://github.com/ChainSafe/lodestar/pull/2816, I don't see the reason for this. The error can still happen (https://github.com/ChainSafe/lodestar/issues/6021#issuecomment-1766438390)  and should be ignored.